### PR TITLE
Add parameter to enable JIT compilation of PCRE regexps

### PR DESCRIPTION
**Pull request by dud225**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -110,3 +110,5 @@
 default['nginx']['ohai_plugin_enabled'] = true
 
 default['nginx']['load_modules'] = []
+
+default['nginx']['pcre_jit'] = 'off'
```
Comment (Xorima) [inline attributes/default.rb]: Would this not be better as a true false?

Comment (dud225) [reply attributes/default.rb]: There are other parameters that are using the `on`/`off` values, but well I've updated my code to follow your suggestion.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,7 @@ Generally used attributes. Some have platform specific values. See `attributes/d
 - `node['nginx']['sts_max_age']` - Enable Strict Transport Security for all apps (See: <http://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security>). This attribute adds the following header: Strict-Transport-Security max-age=SECONDS to all incoming requests and takes an integer (in seconds) as its argument.
 - `node['nginx']['default']['modules']` - Array specifying which modules to enable via the conf-enabled config include function. Currently the only valid value is "socketproxy".
 - `node['nginx']['load_modules']` - Array of paths to modules to dynamically load on nginx startup using the `load_module` directive. Default is `[]`.
+- `node['nginx']['pcre_jit']` - Enables or disables the use of “just-in-time compilation” (PCRE JIT) for the regular expressions known by the time of configuration parsing. . Default is off.
```
Comment (Xorima) [inline README.md]:
```suggestion
- `node['nginx']['pcre_jit']` - Enables or disables the use of “just-in-time compilation” (PCRE JIT) for the regular expressions known by the time of configuration parsing. . Default is false (off).
```

Comment (dud225) [reply README.md]: Sorry I missed that one.

**Pull request merged**